package.json

```diff
@@ -19,7 +19,7 @@
   "main": "./lib",
   "dependencies": {
     "buffer-writer": "1.0.1",
-    "generic-pool": "2.1.1",
+    "generic-pool": "git://github.com/Shyp/node-pool.git#shyp-master",
```

Comment:
Would you be opposed to waiting for the upstream fixes to `generic-pool` to land in npm before we merge this? Alternatively you could publish your fork at a different module name on npm?  I really don't want to rely on github for installs - this breaks some production workflows and is especially painful whenever github goes down.  What do you think?